README.md

```diff
@@ -20,10 +20,10 @@ and then relate the `kubernetes-e2e` charm.
 
 ```shell
 juju deploy kubernetes-core
-juju deploy cs:~containers/kubernetes-e2e
-juju add-relation kubernetes-e2e:kube-control kubernetes-control-plane:kube-control
-juju add-relation kubernetes-e2e:kubernetes-master kubernetes-control-plane:kube-api-endpoint
-juju add-relation kubernetes-e2e easyrsa
+juju deploy ./kubernetes-e2e_ubuntu-22.04-amd64.charm kubernetes-e2e
```

Comment:
let's keep this referencing the charm from the store...

```suggestion
juju deploy kubernetes-e2e
```

Reply:
@eaudetcobello can you return this to how it was?  Rather than reference a locally built file